<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <rosparam param="/marker_publisher/marker_list"> [11, 12, 13, 15] </rosparam>

    <node pkg="rosbot_bringup" type="camera.py" name="camera" output="screen" launch-prefix="xterm -e">
    </node>

    <node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher" output="screen" launch-prefix="xterm -e">
        <remap from="/image" to="/camera/color/image_raw"/>
    </node>
    
    <node pkg="image_view" type="image_view" name="image_view" output="screen">
        <remap from="image" to="/aruco_marker_publisher/result"/>
    </node>

</launch>

